### PR TITLE
blog: meet our contributors APAC India region

### PR DESCRIPTION
**Pull request by Priyankasaggu11929**
This PR adds the first blog post for India region, in the series of ["Meet Our Contributors - APAC"](https://docs.google.com/document/d/1zB-FOvfbnKbiWlGgInnDB06hDfLsTQuuKsXQCqc-lL0/edit#heading=h.yr3qa2ko7xq) blog series.

Reference issue: https://github.com/kubernetes/community/issues/4469#issuecomment-942422348

### CODE REVIEW

**Review by Debanitrkl — COMMENTED** [review]
LGTM 🎉🎉

**Comment by sftim** [comment]
Hi folks

**Comment by Priyankasaggu11929** [comment]
Hello @sftim, Happy New Year :)

And thanks for pointing. I've raised the asked PR: https://github.com/kubernetes/contributor-site/pull/279

**Comment by sftim** [comment]
/hold
for kubernetes/contributor-site#279

(and then we should synchronize the copy, publication date, etc)

**Comment by Priyankasaggu11929** [comment]
@sftim, I've synchronized this PR with the reviewed content & publication date (Jan 10, 2022) from the kubernetes/contributor-site#279

Kindly please review. Thank you!

File: content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (India region)"
+date: 2022-01-10
+slug: meet-our-contributors-india-ep-01
```
Comment (sftim) [inline content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md:5]:
```suggestion
layout: blog
title: "Meet Our Contributors - APAC (India region)"
date: 2022-01-10T12:00:00+0000
slug: meet-our-contributors-india-ep-01
canonicalUrl: https://kubernetes.dev/blog/2022/01/10/meet-our-contributors-india-ep-01/
```

(delay until the other post is published, and mark that one as the canonical URL)

**Review by sftim — COMMENTED** [review]
Tiny remaining changes

File: content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md (hunk 1 above)
Comment (Priyankasaggu11929) [reply content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md]: @sftim, looks like the [other post](https://kubernetes.dev/blog/2022/01/10/meet-our-contributors-india-ep-01/) is live on the site now. I've updated the PR with the changes pointed above.

Comment (sftim) [reply content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md]: Oops, that's a bug!

Comment (Priyankasaggu11929) [reply content/en/blog/_posts/2022-01-10-meet-our-contributors-APAC-India-region-01.md]: Yea, I was wondering the same. 😅 The date for publish is Jan 10.

**Pull request merged**